docs/registry.md

```diff
@@ -34,11 +34,12 @@ Proposals can be of different type and include (`proposal_metadata`) different d
 2. Proposal to update the set of successful-proposal receiver contract addresses.
 This proposal takes a parameter that has two constructors. The proposal can add to,
 or remove from the set of addresses.
-3. Transfer proposals, that can transfer XTZ or tokens as well as update the
+3. Transfer proposals, that can transfer XTZ or tokens (FA2 as well as FA1.2) as well as update the
```

Comment:
A few things here seem to be incorrect

For once, one can transfer a list of XTZ, FA2 and FA1.2 tokens in a single proposal AFAIK. It seems that currently this is phrased as one having to pick between:
- an XTZ transfer
- a list of FA2 transfers
- a single FA1.2 transfer

Reply:
Maybe the original text could have been improved, but I meant that the changes introduced seem incorrect.

The original text looked something like this:
```
Transfer proposals [...] This includes:
- a list of `update` items [...]
- a list of items where each item contains: [... long_type_here ...] The left part is used for XTZ transfers, the right part is used for FA2 transfers.
- `nat %agoraPostID`  [...]
```
Reading this I'd think that each transfer proposal contains 3 fields, one of which is a list of transfers, of which each item can be of two different types (which was correct AFAIK).

With the edit here, the text looks something like this:
```
Transfer proposals [...] This includes:
- a list of `update` items [...]
- for FA2 transfers a list of items where each item contains:  [... long_type_here ...] The left part is used for XTZ transfers, the right part is used for FA2 transfers.
  for FA1.2 transfers a single transfer item represented [...]
- `nat %agoraPostID`  [...]
```
here it's not longer clear:
- if FA1.2 transfers will be in the same list
- why the long type that contains two types of transfers doesn't contain all 3 and why it is introduced as "for FA2 transfers" if it also used for XTZ ones?
- can there be only one FA1.2 transfer? Are only FA2 ones in a list?

Reply:
Ok, makes sense, I have revised that section now.